src/include/function/vector_operations.h

```diff
@@ -16,6 +16,7 @@ using scalar_exec_func = std::function<void(
     const std::vector<std::shared_ptr<common::ValueVector>>&, common::ValueVector&)>;
 using scalar_select_func = std::function<bool(
     const std::vector<std::shared_ptr<common::ValueVector>>&, common::SelectionVector&)>;
+using operation_vectors = std::vector<std::unique_ptr<VectorOperationDefinition>>;
```

Comment:
Let's not use operation_vectors because it's make people it's a vector, try to use something along the line of 
`vector_operation_definitions` or `vector_operation_collection`